test/common/dns.js

```diff
@@ -287,4 +287,51 @@ function writeDNSPacket(parsed) {
   }));
 }
 
-module.exports = { types, classes, writeDNSPacket, parseDNSPacket };
+
+const addresses = {
```

Comment:
I am not quite sure if `common/dns` is the best place to put this. Also pending docs.

Reply:
If it’s specific to `test/internet`, it might be better live in a file there (similar to what `test/async-hooks` has)?

Reply:
I'm not 100% sure about that... One the one hand, I'd rather see all "harness" code in `common`, on the other hand this should not be used by any other suite.
/ping @Trott & @nodejs/testing

Reply:
Alternatively just add another module, `test/common/internet.js`

Reply:
> I'm not 100% sure about that... On the one hand, I'd rather see all "harness" code in common, on the other hand this should not be used by any other suite.

I think it's not a terrible bad idea to replace other `google.com` references in parallel tests..also when my ISP DNS hijacks `NXDOMAIN` there are constant errors from `test-net-connect-immediate-finish` and `test-net-better-error-messages-port-hostname`, it would be helpful to replace those `this.hostname.is.invalid` with `***` configured via environment variables.